### DISCUSSION TITLE
Feature request: Allow to load remote images in lottie files

### DISCUSSION THREAD

**Discussion by Kiryous** [airbnb/lottie-ios#1941] [Ideas]
## Use case
I have a generic lottie animation with placeholder image. I want to replace placeholder with real image in runtime with some remote image from CDN.

## Desired behavior
I could use remote images URLs in lottie assets section.
```
  "assets": [
    {
      "id": "image_0",
      "w": 185,
      "h": 246,
      "u": "",
      "p": <remoteURL>
      }
]
```

## Current behavior
Right now the only way to accomplish that is to convert image to base64 and replace the asset url in lottie json.

**Comment by calda** [comment]
Are you familiar with [dotLottie](https://dotlottie.io/) files? They can directly include image files that are used in the animation, without having to encode them as base 64. imo that's the best way to handle images in Lottie animations, in cases where you know the image content up-front and don't have to change it dynamically at runtime.

**Reply by Kiryous** [reply]
hey @calda! thank you for the promt reply. dotLottie is perfect for static animations, but I do need to swap image on runtime

**Reply by ITstollman** [reply]
found a solution for cdn images in dotlottie?

**Comment by NikKovIos** [comment]
@Kiryous check this https://github.com/airbnb/lottie-ios/discussions/1977
